tests/README.md

```diff
@@ -2,25 +2,20 @@
 To ensure the whole stack is working as expected and track bugs, a layer based test architecture is needed on OCP. This plan consists of four layers. The first and second layers mentioned below, would be a part of the future automation testing framework on each OCP (x,y,z) release.
 
 #### L1 First Layer: Host OS and Driver Interfaces 
-This layer's test cases will be executed after the cluster is setup, NFD operator is installed and right before the Intel Device Plugins Operator installation. 
-* Test cases for in-tree and OOT tree drivers
+This layer's test cases will be executed after [KMMO](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/kmmo/README.md), [machine configuration](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/machine_configuration/README.md) and [NFD operator](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/nfd/README.md) is installed and right before the [Intel Device Plugins Operator](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/device_plugins/README.md) installation. Refer to [readme](https://github.com/intel/intel-technology-enabling-for-openshift/blob/main/tests/l1/README.md).
```

Comment:
Layer 1 test cases should be executed before [Deploy Intel Device Plugins on OpenShift](/device_plugins/README.md#deploy-intel-device-plugins-on-openshift) and after deploying OOT drivers like [Intel Data Center GPU Driver on OpenShift](/kmmo/README.md#managing-intel-dgpu-driver-with-kmm-operator)

Reply:
do we exclude mentioning kmmo, nfd and mco?